### PR TITLE
4 ticket screen

### LINKED ISSUE #4: Create Ticket Screen
Create ticket screen

### PR DESCRIPTION
**Pull request by caudaganesh**
Closes #4 
#8 

- [x] add ticket screen
- [x] add main screen
- [x] add ticket list
screenshot_2017-10-02-22-57-58-475_io devsummit android

### CODE REVIEW

File: app/src/main/java/io/devsummit/devsummit/Activities/LoginActivity.java [hunk 1]
```diff
@@ -81,12 +80,12 @@
     private EditText mEmailView;
     private EditText mPasswordView;
     private View mProgressView;
-    private View mLoginFormView;
     private Button mLoginButton;
     private Button mLoginPhoneButton;
 
     @Override
     protected void onCreate(Bundle savedInstanceState) {
+        authHelper = new User_Authentication_Helper(this);
```
Comment (mgufrone) [inline app/src/main/java/io/devsummit/devsummit/Activities/LoginActivity.java]: why are you using this kind of case? Java usually just use CamelCase without any underscore. Except it is property. @caudaganesh

**Review by mgufrone — CHANGES_REQUESTED** [review]
can we change the namespace into io.devsummit.native-android?

File: app/src/main/java/io/devsummit/devsummit/Activities/LoginActivity.java (hunk 1 above)
Comment (caudaganesh) [reply app/src/main/java/io/devsummit/devsummit/Activities/LoginActivity.java]: okay i will fix it sir

File: app/src/main/java/io/devsummit/devsummit/ViewAdapters/NotificationViewAdapter.java [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+    public static class NotificationViewHolder extends RecyclerView.ViewHolder {
+        // TODO: whatever views you need to bind
+        public TextView messageTextView;
+        public TextView titleTextView;
+        public TextView createdTextView;
+        public TextView senderTextView;
+
+        public NotificationViewHolder(View v) {
+            super(v); // done this way instead of view tagging
+            messageTextView = (TextView) v.findViewById(R.id.message_text);
```
Comment (mgufrone) [inline app/src/main/java/io/devsummit/devsummit/ViewAdapters/NotificationViewAdapter.java]: Try to explore about Butterknife for dependency injection in view @caudaganesh

File: app/src/main/java/io/devsummit/devsummit/Remote/APIService.java [hunk 3]
```diff
@@ -31,4 +37,19 @@
     @POST("auth/register")
     @Headers("Content-Type:application/json")
     Call<RegisterModel> register(@Body RegisterCredentials credentials);
+
+    @GET("api/v1/notifications?page=1")
```
Comment (mgufrone) [inline app/src/main/java/io/devsummit/devsummit/Remote/APIService.java]: if you change the page into another page, what happen? I'm not sure this one is properly set

**Comment by caudaganesh** [comment]
OK, I will try to change it sir

**Comment by mgufrone** [comment]
@caudaganesh let me know if this is ready to be reviewed.

**Comment by caudaganesh** [comment]
it is ready sir, already put apk on fabric as well. thank you

**Comment by mgufrone** [comment]
Okay I will review it again @caudaganesh

**Review by mgufrone — APPROVED** [review]
This is looking good.

**Pull request merged**